ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -239,6 +239,107 @@ device_classes: !ZenPackSpec
                                 rrdmin: 0
                             missing:
                                 rrdmin: 0
+                graphs:
+                    Utilization:
+                        units: percent
+                        miny: 0
+                        maxy: 100
+                        graphpoints:
+                            size:
+                                dpName: volume_size
+                                lineType: DONTDRAW
+                            free:
+                                dpName: volume_free
+                                lineType: AREA
+                                rpn: "size,-,size,/,-100,*"
+                thresholds:
+                    unallocatable:
+                        dsnames: [status_allocatable]
+                        minval: 1
+                        eventClass: /Status
+                    exported:
+                        dsnames: [status_exported]
+                        maxval: 0
+                        severity: 2
+                        eventClass: /Status
+                    missing:
+                        dsnames: [status_missing]
+                        maxval: 0
+                        eventClass: /Status
+            VolumeGroup:
+                description: Volume Group Monitoring
+                targetPythonClass: ZenPacks.zenoss.LinuxMonitor.VolumeGroup.VolumeGroup
+                datasources:
+                    group:
+                        type: COMMAND
+                        commandTemplate: '/usr/bin/sudo /usr/sbin/vgs --noheadings --units b --nosuffix -o vg_name,vg_size,vg_free'
+                        usessh: true
+                        parser: ZenPacks.zenoss.LinuxMonitor.parsers.linux.utilization
+                        datapoints:
+                            size:
+                                rrdmin: 0
+                            free:
+                                rrdmin: 0
+                    status:
+                        type: COMMAND
+                        commandTemplate: "/usr/bin/sudo /usr/sbin/vgs --noheadings -o vg_name,vg_attr | awk '{split($$2, chars, \"\"); printf(\"%s \", $$1); for (i=1;i<=3;i++) printf(\"%d \", chars[i]==\"-\" ? 0 : 1); printf(\"\\n\")}'"
```

Comment:
Remove the absolute path to `vgs`. It isn't in /usr/sbin/ on Ubuntu. Also, it'd probably be a better idea to use the parser to parse than to pipe the output into awk to do some preprocessing then have the parser further process the results.